wiki/Community/Projects/fr.md

```diff
@@ -1,85 +1,92 @@
 ---
-outdated: true
+outdated: true # À jour par rapport à `en.md`
```

Comment:
@Walavouchey this comment say that its "up to date compare to en.md", im not really with the idea of comment this part because if the en.md is updated this comment will still be here and will be inappropriate. In the past days you told about a new tag for that if im not wrong, did you think it will available soon ?

Reply:
that was a just a suggestion, unsure when that might happen. this will be fine for now